### ISSUE TITLE
Annotations are ignored when running phpunit with selenium

### ISSUE THREAD

**Issue by cryptton2004** [allure-framework/allure-phpunit#28]
Hello,

I am using phpunit with selenium on windows 7 to automate my functionality tests.
As an example, I have the following structure in my featureClassTest.php:
```
class featureClassTest extends somethingThatEventuallyExtendsPHPUnit_Extensions_SeleniumTestCase {

    public function testCase()
    {
        $this->executeStep("This is step one", function () {
            $this->stepOne();
        });

        $this->executeStep("This is step two", function () {
            $this->stepTwo();
        });
    }
...
}
```

The problem is that (depending on the browser I'm using), phpunit will fire the test something like this:
```featureClassTest: Firefox```
So, in the phpunit AllureAdapter.php, if I make a ``` var_dump($suiteName)``` in ```public function startTestSuite(PHPUnit_Framework_TestSuite $suite) {}```, I will get ```featureClassTest: Firefox```.
The following will now return false:
```if (class_exists($suiteName, false)) {
            ...
        }```
I made a workaround this issue for my particular case, but I'm sure it can be generalized by using something like browser['name']. This is how it currently looks for my case:
```
    public function startTestSuite(PHPUnit_Framework_TestSuite $suite)
    {
        $suiteName = $suite->getName();
        $suiteName = str_replace(": Firefox", "",$suiteName);
        $event = new TestSuiteStartedEvent($suiteName);
        $this->uuid = $event->getUuid();
        $this->suiteName = $suiteName;

        $suiteName = str_replace(": Firefox", "",$suiteName);
        var_dump($suiteName);
        if (class_exists($suiteName, false)) {
            $annotationManager = new Annotation\AnnotationManager(
                Annotation\AnnotationProvider::getClassAnnotations($suiteName)
            );
            $annotationManager->updateTestSuiteEvent($event);
        }

        Allure::lifecycle()->fire($event);
    }
```

**Comment by tyz910** [comment]
possibly fixed in #32 - $suiteName not used anymore

**Comment by cryptton2004** [comment]
I confirm that this fix worked for me. Thank you :)
